src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import MyCalculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <MyCalculator />
     </div>
   );
```

Comment:
- [ ] Please consider changing this functional component to a `class` component, the project requirements for this project require that all components should be class-based components. Please see the screenshot below.

![Screenshot from 2022-11-30 19-25-19](https://user-images.githubusercontent.com/16392046/204853306-45ba7d18-7f30-47df-bd1d-bf3e159af46f.png)

You can do this as shown below
```JavaScript
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

Reply:
Thank you for your review, it help me very much to understand better! 👍